rundeckapp/src/integration-test/groovy/rundeck/services/ExecutionServiceIntegrationSpec.groovy

```diff
@@ -0,0 +1,4 @@
+package rundeck.services
```

Comment:
This file seems unnecessary. Can we remove it?

Reply:
Hah, yes.. That was abandoned. It seemed like a good idea and turned into a massive PITA.